packages/manager/apps/pci-users/src/components/users/RCloneDownloadModal.tsx

```diff
@@ -0,0 +1,235 @@
+                  ></OsdsIcon>
+                </span>
+              </OsdsLink>
+              <OsdsFormField className={'mt-2'}>
+                <OsdsText
+                  slot="label"
+                  level={ODS_THEME_TYPOGRAPHY_LEVEL.button}
+                  color={ODS_THEME_COLOR_INTENT.text}
+                >
+                  {t(
+                    'pci_projects_project_users_download-rclone_file_type_label',
+                  )}
+                </OsdsText>
+                <OsdsRadioGroup
+                  required={true}
+                  className={'flex mb-4'}
+                  name={'filetype-radiogroup'}
+                  value={fileType}
+                  onOdsValueChange={handleFileTypeChanged}
+                  data-testid="downloadFileType"
+                >
+                  <OsdsRadio
+                    value={DOWNLOAD_FILETYPE.SWIFT}
+                    name={'filetype-radiogroup'}
+                    className={'mr-4'}
+                    color={ODS_THEME_COLOR_INTENT.primary}
+                  >
+                    <OsdsRadioButton
+                      size={ODS_RADIO_BUTTON_SIZE.sm}
+                      color={ODS_THEME_COLOR_INTENT.primary}
+                    >
+                      <OsdsText
+                        slot={'end'}
+                        size={ODS_THEME_TYPOGRAPHY_SIZE._500}
+                        color={ODS_THEME_COLOR_INTENT.text}
+                      >
+                        {DOWNLOAD_FILETYPE.SWIFT}
+                      </OsdsText>
+                    </OsdsRadioButton>
+                  </OsdsRadio>
+                  <OsdsRadio
+                    value={DOWNLOAD_FILETYPE.S3}
+                    name={'filetype-radiogroup'}
+                  >
+                    <OsdsRadioButton
+                      size={ODS_RADIO_BUTTON_SIZE.sm}
+                      color={ODS_THEME_COLOR_INTENT.primary}
+                    >
+                      <OsdsText
+                        size={ODS_THEME_TYPOGRAPHY_SIZE._500}
+                        color={ODS_THEME_COLOR_INTENT.text}
+                        slot={'end'}
+                      >
+                        {DOWNLOAD_FILETYPE.S3}
+                      </OsdsText>
+                    </OsdsRadioButton>
+                  </OsdsRadio>
+                </OsdsRadioGroup>
+              </OsdsFormField>
+              <OsdsFormField>
+                <OsdsText
+                  slot="label"
+                  level={ODS_THEME_TYPOGRAPHY_LEVEL.heading}
+                  color={ODS_THEME_COLOR_INTENT.text}
+                >
+                  {t(
+                    'pci_projects_project_users_download-rclone_region_label_global',
+                  )}
+                </OsdsText>
+                {fileType === DOWNLOAD_FILETYPE.S3 ? (
+                  <S3StorageRegions
+                    projectId={projectId}
+                    onS3StorageRegionChange={onRegionChange}
```

Comment:
you can pass "setRegion" directly instead of "onRegionChange" and remove the onRegionChange function